config/environments/development.rb

```diff
@@ -27,7 +34,10 @@
   else
     config.action_controller.perform_caching = false
 
-    config.cache_store = :null_store
+    cache_ssl = (ENV.fetch('CACHE_SSL') { 'false' }) == 'true'
```

Comment:
Should this be in the other path? It looks like it is enabling caching to me...

Reply:
This is just setting the SSL nature of the cache.  My reading of that is that you either have in memory cache, or if enabled, redis.  Removed the comment